lib/person.rb

```diff
@@ -0,0 +1,24 @@
+require './lib/craft'
+
+class Person
+
+  attr_reader :name, :interests, :supplies
+
+  def initialize(data)
+    @name = data[:name]
+    @interests = data[:interests]
+    @supplies = {}
+  end
+
+  def add_supply(supply_name, count)
+    @supplies[supply_name] ||= 0
+    @supplies[supply_name] += count
+  end
+
+  def can_build?(craft)
+    craft.supplies_required.map do |supply, count|
+      @supplies.keys.include?(supply) && @supplies[supply] >= count
```

Comment:
Again, we want to be conscious of the use of symbols vs strings

```suggestion
      @supplies.keys.include?(supply.to_sym) && @supplies[supply.to_sym] >= count
```